code/controllers/subsystem/adjacent_air.dm

```diff
@@ -1,6 +1,5 @@
 SUBSYSTEM_DEF(adjacent_air)
 	name = "Atmos Adjacency"
-	flags = SS_BACKGROUND
 	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
```

Comment:
@LemonInTheDark I dunno how important Atmos is but I assume it is so I removed this but if you feel its fine with it tell me